packages/core/bridge/providers/cctp/cctpUtils.ts

```diff
@@ -153,6 +153,11 @@ export function getMessageTransmitterFromChain(chain: string) {
             abi: baseMessageTransmitterAbi,
             address: CCTP_EVM_CONTRACTS.V1.MESSAGE_TRANSMITTER.baseSepolia as Address,
         }
+    } else if (chain === 'ethereumSepolia') {
+        return {
+            abi: baseMessageTransmitterAbi,
```

Comment:
Prob worth renaming to `evmMessageTransmitterAbi` now, assuming this is working for eth + base.